docs/en/operations/system-tables/zookeeper_connection.md

```diff
@@ -11,7 +11,8 @@ Columns:
 -   `host` ([String](../../sql-reference/data-types/string.md)) — The hostname/IP of the ZooKeeper node that ClickHouse connected to.
 -   `port` ([String](../../sql-reference/data-types/string.md)) — The port of the ZooKeeper node that ClickHouse connected to.
 -   `index` ([UInt8](../../sql-reference/data-types/int-uint.md)) — The index of the ZooKeeper node that ClickHouse connected to. The index is from ZooKeeper config.
--   `connected_time` ([String](../../sql-reference/data-types/string.md)) — When the connection was established
+-   `connected_time` ([DateTime](../../sql-reference/data-types/datetime.md)) — When the connection was established
+-   `session_uptime_elapsed_seconds` ([UInt64](../../sql-reference/data-types/int-uint.md)) — When the connection was elapsed seconds
```

Comment:
```suggestion
-   `session_uptime_elapsed_seconds` ([UInt64](../../sql-reference/data-types/int-uint.md)) — Seconds elapsed since the connection was established
```